use/currencies.js

```diff
@@ -0,0 +1,56 @@
+const EUR = '€' // Euro
+const USD = '$' // US Dollar
+const BDT = '৳' // Bangladeshi Taka
+const CRC = '₡' // Costa Rican Colón
+const ETH = 'Ξ' // Ethereum
+const GBP = '£' // British Pound Sterling
+const GEL = 'ლ' // Georgian Lari
+const ILS = '₪' // Israeli New Sheqel
+const INR = '₹' // Indian Rupee
+const JPY = '¥' // Japanese Yen
+const KRW = '₩' // South Korean Won
+const LTC = 'Ł' // Litecoin
+const NGN = '₦' // Nigerian Naira
+const PHP = '₱' // Philippine Peso
+const PLN = 'zł' // Polish Zloty
+const PYG = '₲' // Paraguayan Guarani
+const RUB = '₽' // Russian Ruble
+const THB = '฿' // Thai Baht
+const TRY = '₺' // Turkish Lira
+const UAH = '₴' // Ukrainian Hryvna
+const VND = '₫' // Vietnamese Dong
+const XMR = 'ɱ' // Monero
+const BTC = '₿' // Bitcoin
```

Comment:
`10 EUR` is `0,000023 BTC`, the minimum you can enter now is `0,01 BTC`. Not sure how to solve it.